expression/integration_test.go

```diff
@@ -7598,3 +7598,45 @@ func TestCastJSONTimeDuration(t *testing.T) {
 		fmt.Sprintf("6 %s %s 12:12:12 12:12:12 TIME", nowDate, nowDate),
 	))
 }
+
+func TestRegexpPushdown(t *testing.T) {
```

Comment:
if you have extra time, you can refactor this test to make the test case separate from test code, like other tests. Which will make it easy to regenerate tests result.